### ISSUE TITLE
Should add UILabel inset option

### ISSUE THREAD

**Issue by dzpt** [tbaranes/FittableFontLabel#1]
Great solution.
I've been struggling with fitable uilabel and NSAttributedString.
But i think you should provide inset option, because some kind of font will lose it apart if we leave inset = 0 and alignment = right.

Thanks

**Comment by tbaranes** [comment]
Hey! Interesting solution, but I'm not sure about it. I'd like to keep the usage as simple as possible by limiting the real customizations to the attributed string options.

To simulate the insets, embeds your `UILabel` in an `UIView` wont be enough? Or just playing with the rect size?

**Comment by dzpt** [comment]
I've modified your code to add inset to the `UILabel`, putting `UILabel` in an `UIView` will waste the memory. And in some cases, `UILabel` without inset can mis-display some part of the text like these example.

![simulator screen shot apr 15 2016 3 03 06 am](https://cloud.githubusercontent.com/assets/5781080/14541939/9f80cad8-02b6-11e6-8090-abbec6536916.png)

**Comment by tbaranes** [comment]
Got it. PR welcome :+1:

**Comment by tbaranes** [comment]
Thinking back to your issue, using `rectSize` from `fontSizeToFit` should solve your issue since you can pass a smaller rect than the real bounds of your `UILabel`. So, I suppose you get that issue because you are using directly the `UILabel` subview and not the label extension?

If that so, I have a better understand of your needs. We could add an inset inspectable in `FittableFontLabel` in order to have custom insets. Do you think that could solve that issue?

**Comment by tbaranes** [comment]
@Tom29 I implemented this feature in #5. I will merge it tonight and release a new cocoapods version. You will find four new property in FittableFontLabel:

``` swift
    @IBInspectable public var leftInset: CGFloat = 0
    @IBInspectable public var rightInset: CGFloat = 0
    @IBInspectable public var topInset: CGFloat = 0
    @IBInspectable public var bottomInset: CGFloat = 0
```

I also updated the example project.
Let me know if you have any questions.
